packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/chat_wrappers.py

```diff
@@ -371,9 +375,16 @@ def _set_completions(span, choices):
             span, f"{prefix}.finish_reason", choice.get("finish_reason")
         )
 
+        if choice.get("content_filter_results"):
+            _set_span_attribute(
+                span, f"{prefix}.content_filter_results",
```

Comment:
let's extract this to a const as well

Reply:
Did that, but the constant is now in the scope of the file. Are we considering moving it somewhere else?